src/lib/pluginUtils.ts

```diff
@@ -17,14 +17,47 @@ import * as moment from "moment-timezone";
 
 import { getArc } from "../arc";
 
-export const getRewarderName = (pluginToRegisterData: string): string => {
+
+/**
+ * @param {string} pluginName Plugin name
+ * @param {string} pluginToRegisterData Plugin encoded data
+ * @returns {any} An object containing voting params, real plugin name and contract to call if acceptable
+ */
+export const decodePluginToRegisterData = (pluginName: string, pluginToRegisterData: string): any => {
   const WEB3 = require("web3");
   const web3 = new WEB3();
+  const PLUGIN_PARAMS = require("./plugin_params.json");
```

Comment:
does this `require` inside function scope is following our code standard ?
should it be moved to the beginning of the file ?

Reply:
Here https://github.com/daostack/alchemy/pull/1865 you to suggested to use it as a local var. You mean only the .json file `require` should be outside?

Reply:
I meant only  for `const web3 = new WEB3(); `
the `require`  should be outside.
Also WEB3 -> Web3